src/test/java/com/google/api/codegen/testdata/go_doc_library.baseline

```diff
@@ -19,6 +19,8 @@
 // library API.
 //
 // A simple Google Example Library API.
+//
+// Use the handwritten client (cloud.google.com/go/library) in preference to this.
```

Comment:
I think we shouldn't use the word "handwritten". 

Instead of trying to find the perfect word, let's just say "Use the client at {path} in preference to this."